practices/managing-static-assets.md

```diff
@@ -0,0 +1,16 @@
+# Managing static assets
+
+Static assets are files that don't need to be generated, modified or processed by an application before being delivered to the user. This typically includes CSS, JavaScript, images and web fonts.
+
+This content rarely changes and its contents don't depend on user input or preferences.
+
+For this kind of static resources it is our choice to **self-host them on our servers** instead of including them on our pages from an external CDN.
+
+## Benefits of self-hosting static assets
+
+* **Resilience.** As long as our site is up, the resource will be up. We don't depend on other servers or additional CDNs and we won't be affected by their outages, or closure of the service.
+* **Consistency.** Static assets are cached and served in exactly the same way that the reset of our resources are, and they use the same CDN. As a result, timings and user experience are more consistent all around the world.
+* **Security.** reduces our front-end [attack surface](https://en.wikipedia.org/wiki/Attack_surface).
+* **Performance.** Self-hosting can save us the cost of the DNS request, TCP connection and TLS negotiation, which in slow connections can easily be between 1000ms and 2000ms. This can also be mitigated with `rel=preconnect`.
+* **Filtering.** Some CDNs are filtered by the big firewall in China. This can slow down the loading of the page, or even make them break. Self-hosting static assets prevents this.
+* **Security.** Self-hosting the asset makes it easier for us to use [Subresource Integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) in the future.
```

Comment:
Should this form a sub-list under the other Security point, above? Also suggest removing "in the future"